File: consumer/aisl-consumer-suite/packages/aisl-sdk/examples/list_systems.py
import argparse
from aisl_sdk import AislClient

p = argparse.ArgumentParser()
p.add_argument("--api-url", required=True)
a = p.parse_args()

with AislClient(a.api_url) as client:
    for system in client.list_systems():
        print(system.system_id, system.active_revision_id)

File: consumer/aisl-consumer-suite/packages/aisl-sdk-typescript/examples/list-systems.mjs
import {AislClient} from '../dist/index.js';
const apiUrl = process.argv[2];
if (!apiUrl) throw new Error('usage: node examples/list-systems.mjs <api-url>');
const client = new AislClient(apiUrl);
for (const system of await client.listSystems()) console.log(system.system_id, system.active_revision_id);
